modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/140_data_stream_aliases.yml

```diff
@@ -240,7 +240,8 @@
             test: {}
 
   - do:
-      indices.get_alias: { }
+      indices.get_alias:
+        index: test*
```

Comment:
Fetching aliases only for `test` indices to avoid getting `.security` one.

Reply:
No, it's just how our YAML tests are setup. For simplicity, they all use the same user with the `_es_test_root` role that grants access to all indices (including system). I've tried to explain this in this comment https://github.com/elastic/elasticsearch/pull/120323#issuecomment-2612195536.